packages/client/package.json

```diff
@@ -23,7 +23,7 @@
     "docs:build": "typedoc --tsconfig tsconfig.prod.json",
     "lint": "ethereumjs-config-lint",
     "lint:fix": "ethereumjs-config-lint-fix",
-    "tape": "tape -r ts-node/register",
+    "tape": "TS_NODE_FILES=true tape -r ts-node/register",
```

Comment:
interestingly enough I found this was needed for `tape -r ts-node/register` to include the `typings` folder as defined in `tsconfig.json:include`

Reply:
Ugh. 😄 

👍